src/parser/tests.rs

```diff
@@ -1,45 +1,6 @@
-use std::fmt;
+use crate::runtime::divert::PushPopType;
 
-use serde::de::{Deserialize, Deserializer, MapAccess, Visitor};
-
-// TODO
-struct ListDefinitions {}
-
-struct ListDefinitionsVisitor {}
-
-impl ListDefinitionsVisitor {
-    fn new() -> Self {
-        ListDefinitionsVisitor {}
-    }
-}
-
-impl<'de> Visitor<'de> for ListDefinitionsVisitor {
-    // Our Visitor is going to produce a RuntimeGraph.
-    type Value = ListDefinitions;
-
-    // Format a message stating what data this Visitor expects to receive.
-    fn expecting(&self, formatter: &mut fmt::Formatter) -> fmt::Result {
-        formatter.write_str("List definitions")
-    }
-
-    fn visit_map<A>(self, _map: A) -> Result<Self::Value, A::Error>
-    where
-        A: MapAccess<'de>,
-    {
-        Ok(ListDefinitions {})
-    }
-}
-
-impl<'de> Deserialize<'de> for ListDefinitions {
-    fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
-    where
-        D: Deserializer<'de>,
-    {
-        // Instantiate our Visitor and ask the Deserializer to drive
-        // it over the input data, resulting in an instance of ListDefinitions.
-        deserializer.deserialize_map(ListDefinitionsVisitor::new())
-    }
-}
+use super::*;
 
 #[cfg(test)]
 mod tests {
```

Comment:
Does this part compile still? Better to move the `#[cfg(test)]` to the `mod tests;` in `parser/mod.rs` and get rid of this extra module declaration, otherwise you'll get a `tests` module within the `tests` module.

Reply:
You can see in the github action it's saying
```
running 35 tests
test parser::tests::tests::choice_test ... ok
test parser::tests::tests::container_length_test ... ok
```
With `tests::tests`.